package.json

```diff
@@ -1,43 +1,43 @@
 {
-  "name": "@qonsoll/translation",
-  "version": "0.4.4",
-  "scripts": {
-    "build": " rollup -c",
-    "prepare": "npm run build"
-  },
-  "module": "dist/index.js",
-  "main": "dist/index.cjs.js",
-  "files": [
-    "dist"
-  ],
-  "peerDependencies": {
-    "firebase": ">= 7.0.0 < 9.x",
-    "react": ">= 16.8.0"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.15.8",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-react": "^7.14.5",
-    "@rollup/plugin-babel": "^5.3.0",
-    "@rollup/plugin-commonjs": "^20.0.0",
-    "@rollup/plugin-node-resolve": "^13.0.4",
-    "babel-eslint": "^10.1.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-config-react-app": "^5.2.1",
-    "eslint-plugin-babel": "^5.3.0",
-    "eslint-plugin-flowtype": "^5.1.3",
-    "eslint-plugin-import": "^2.21.2",
-    "eslint-plugin-jsdoc": "^27.0.6",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-react": "^7.20.0",
-    "eslint-plugin-react-hooks": "^4.0.4",
-    "prettier": "^2.0.5",
-    "prop-types": "^15.7.2",
-    "rollup": "^2.55.1",
-    "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-terser": "^7.0.2",
-    "md5": "^2.3.0"
-  }
-}
+	"name": "@qonsoll/translation",
```

Comment:
what happened with spaces here?

Reply:
In reality, i don't now exactly.. Everything seems to be same as in the other places..

<img width="1197" alt="Screenshot 2022-05-20 at 15 16 37" src="https://user-images.githubusercontent.com/101116443/169535969-29116a81-8ff2-48ff-a195-15c36a7cab86.png">

There were an indentation with spaces, but i have changed to the tabs (we have tabs everywhere in this type of files)..